packages/tokens/src/tokens/font/composite.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
Are these not _the_ typography tokens? `composite` feels like very generic naming.

Reply:
Renamed to typography